L13-Python-Interceptors.md

```diff
@@ -0,0 +1,285 @@
+gRPC Python Interceptors
+----
+* Author(s): lidizheng
+* Approver: ericgribkoff
+* Status: In Review
+* Implemented in: Python
+* Last updated: January 17, 2019
+* Discussion at: https://groups.google.com/d/msg/grpc-io/7qIzyCPWbUA/1NKGXAQHCgAJ
+
+## Abstract
+
+This proposal is a hindsight design document that explains the design and implementation details of the interceptor functionality for gRPC Python, which merged in 2017. This gRFC serves as an effort to stabilize the interceptor APIs.
+
+
+## Background
+
+### The Need for Interceptors
+
+Interceptors are middleware which capable of manipulating or reacting to the transporting requests/responses throughout the life cycle of an RPC. It dramatically simplifies the implementation of tracing (See [grpc/grpc#8767](https://github.com/grpc/grpc/issues/8767)) and empowers gRPC developers to add general channel-level business logic.
+
+Interceptors have existed with gRPC for a while now, and its semantics were carried out across gRPC implementations like Java, Go and wrapper languages like Node.js, Ruby.
+
+### Stabilization Status
```

Comment:
You may want to point out that this is the state of interceptors as of January 2019, as this information is likely to change. Alternatively, we may want to leave this information out altogether, as it's not necessary relevant to the design of interceptors in *python* specifically.

Reply:
Yeah, you are right. It helps us understand the situation, but not relevant to the design. Removed.